internal/pkg/service/buffer/store/model/column/column_test.go

```diff
@@ -110,3 +110,120 @@ func TestColumn_Header(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, string(headerMarshalled), val)
 }
+
+func TestColumn_Template_Body_Scalar(t *testing.T) {
+	t.Parallel()
+
+	c := column.Template{Language: column.TemplateLanguageJsonnet, Content: "Body('key1.key2')"}
+
+	val1 := orderedmap.New()
+	val1.Set("key2", "val2")
+	body := orderedmap.New()
+	body.Set("key1", val1)
+	body.Set("key3", "val3")
+
+	header := http.Header{}
+	header.Set("Content-Type", "application/json")
+	header.Set("Content-Encoding", "gzip")
+
+	val, err := c.CsvValue(column.ImportCtx{Body: body, Header: header})
+	assert.NoError(t, err)
+	assert.Equal(t, "\"val2\"", val)
+}
+
+func TestColumn_Template_Body_Object(t *testing.T) {
```

Comment:
Tu chyba teda asi `TestColumn_Template_Body_ArrayOfObjects`

Reply:
🤔 A to mám dát to pole do body nebo kam? Respektive kde to pole mám namodelovat? Body je ordered map.

Reply:
Ano, do body, kde sa moze vyskytnut:
```
body.Set("key1", []any{object1, object2})
```

Ak to chces mat nazornejsie, tak si sprav nejaky JSON a dekoduj ho do OrderedMap
```
{
   "key1": [
      {"foo1": "bar1"},
      {"foo2": "bar2"},
   ]
}
```


-----

Ako som to pisal tak mi napadlo, ze body moze byt aj:
```
[1,2,3]
```
Tam by mohol byt nejaky test `Body([2])`

-----

A este by sme mohli vyriesit, ze co ak chces cele body:
```jsonnnet
{
  fullBody: Body(),
  values: {
    key1: Body('key1')
  }
}
```

Reply:
Preco mozes chciet cele `Body()`? Ak mame na to samostatny typ stlpca?

Napr. chces pouzit nejaku Jsonnet funkciu, napr.
![image](https://user-images.githubusercontent.com/19371734/204525574-3a3a1a19-8085-47b4-b5c5-36cbcf06c9d1.png)

Reply:
![image](https://user-images.githubusercontent.com/19371734/204525916-aa5c686b-ef6c-449f-822e-87fe6c442a84.png)

Reply:
> A este by sme mohli vyriesit, ze co ak chces cele body/ vsetky hlavicky.

A víš jestli tý `jsonnet.NativeFunction` jde vysvětlit že má mít optional parametr? Zřejmě se to validuje podle `NativeFunction.Params` ale to je jen pole stringů, nevidím že by to nějak šlo.
![CleanShot 2022-11-29 at 14 24 14](https://user-images.githubusercontent.com/215660/204540589-b8e29731-ab56-416f-bc54-b337b27c7e68.jpg)

Reply:
A jed das `params` definiciu prazdnu? Tak to spadne pri jednom argumente?

Reply:
jo jo 
![CleanShot 2022-11-29 at 14 31 37](https://user-images.githubusercontent.com/215660/204542106-c1df10ea-5c4e-4ad9-8559-d518dfd5ed5a.jpg)
(Můžeme samozřejmě udělat extra funkci `FullBody()` jen mi přijde divný že to nejde, protože Jsonnet jako takový podle docs optional parametry podporuje.)

Reply:
Pozrel som kod, `NativeFunction`  to neumoznuje, ale kod dalej ma `defaultArg`.
vid `(native *NativeFunction) parameters() []namedParameter {`

Za mna je ok aj `Body("")`.

Reply:
To jsem právě viděl a moc jsem nechápal proč to nejde nastavit. Asi to v Googlu zatím nepotřebovali. 🙂 No, za mě je `Body("")` taky ok. 😄

Reply:
Mozno by davalo zmysel, `Body(), BodyField(...), Headers(), Header()` ... ale to necham na teba, ako to doriesis

Reply:
Jo 👍 upravil jsem. `Header()`/`Headers()` zní fajn, ale u toho body by mi přišlo trochu lepší třeba `Body()`/`FullBody()`.(?) 🙂

Reply:
kludne, hovorim, ze to necham na teba, bol to iba navrh

Reply:
jo, to je pěkný